src/renderer/Renderer.ts

```diff
@@ -538,9 +538,11 @@ abstract class Renderer {
   }
 
   protected _afterRender() {
-    const flicking = getFlickingAttached(this._flicking);
+    if (this._flicking) {
+      const flicking = getFlickingAttached(this._flicking);
 
-    flicking.camera.applyTransform();
+      flicking.camera.applyTransform();
+    }
```

Comment:
Is this included by mistake?

Reply:
I've confirmed that this was included incorrectly. This part will not be released and will be modified.